guides/source/active_support_instrumentation.md

```diff
@@ -70,7 +70,7 @@ ActiveSupport::Notifications.subscribe "process_action.action_controller" do |*a
 end
 ```
 
-You may also pass block with only one argument, it will yield an event object to the block:
+You may also pass a block with only one argument, it will yield an event object to the block:
```

Comment:
Since we're touching this line, would you mind if we also fix the run-on sentence? :pray: 

```suggestion
You may also pass a block that accepts only one argument, and it will receive an event object:
```

Reply:
Yes, I found that sentence a bit clucky as well. 😄 
Would you mind if we make it even simpler?
```suggestion
You may also pass a block, which receives an event object:
```

Reply:
The previous paragraph mentions 'multiple arguments':
> Defining all those block arguments each time can be tedious. You can easily create an ActiveSupport::Notifications::Event from block arguments like this:

So I've changed it to @jonathanhefner 's suggestion.